src/More/LanguageSelection.tsx

```diff
@@ -58,33 +58,35 @@ const LanguageSelection: FunctionComponent = () => {
   }
 
   return (
-    <View style={style.container}>
-      <View style={style.headerContainer}>
-        <GlobalText style={style.headerText}>
-          {t("onboarding.select_language")}
-        </GlobalText>
-        <TouchableOpacity
-          style={style.closeIconContainer}
-          onPress={navigation.goBack}
-        >
-          <SvgXml
-            xml={Icons.XInCircle}
-            fill={Colors.neutral30}
-            width={Iconography.small}
-            height={Iconography.small}
-          />
-        </TouchableOpacity>
+    <>
```

Comment:
Should we have `StatusBar` here?

Reply:
Nope because this screen pops up as a card-style modal -- but I will remove the fragment, good catch!